contracts/scripts/governor/propose.js

```diff
@@ -85,6 +85,27 @@ async function proposeOusdNewGovernorArgs() {
   return { args, description };
 }
 
+// Transfer governance of the OGN Single Asset Staking contract from old to new governor.
```

Comment:
I think for that we need we'll need to add support for a --governorV2 argument to the script.
Similar to this: https://github.com/OriginProtocol/origin-dollar/blob/dc0a7bc396c51a6bee723ab64652f05620004bd9/contracts/scripts/governor/propose.js#L913

But no need to use a different ABI since the governor interface did not change when we did the recent governor upgrade. Something like that:

```
if (config.governorV1) {
  ...
} else if (config.governorV2) {
  const v2GovernorAddress = '0x830622bdd79cc677ee6594e20bbda5b26568b781';
   governor = await ethers.getContractAt("Governor", v2GovernorAddress);
} else {
  governor = await ethers.getContract("Governor");
}
```